pootle/urls.py

```diff
@@ -30,7 +31,15 @@
     url(r'^accounts/evernote/', include('evernote_auth.urls')),
     url(r'^admin/reports/', include('evernote_reports.urls')),
     url(r'', include('evernote_reports.profile_urls')),
+)
+
+if settings.POOTLE_ENABLE_OFFLINE:
```

Comment:
@unho that if you can simply put it in INSTALLED_APPS you don't need a config to POOTLE_ENABLE_OFFLINE